framework/Models/ROM.py

```diff
@@ -259,7 +268,9 @@ def tryStrParse(s):
           continue
         if child.getName() not in self.initializationOptionDict.keys():
           self.initializationOptionDict[child.getName()]={}
-        self.initializationOptionDict[child.getName()][child.value]=child.parameterValues
+        # "tuple" here allows values to be listed, probably not great but works
+        key = child.value if not isinstance(child.value,list) else tuple(child.value)
```

Comment:
I did not get the point here. The type of input is determined by the InputData, why do you need to check the input here?

Reply:
Sometimes "child.value" is a list, which is an unhashable type, so cannot be a dictionary key.  If this is the case, it needs to be cast as a tuple, which is hashable.